records_mover/records/targets/table/move_from_records_directory.py

```diff
@@ -50,12 +50,21 @@ def load_plan(self) -> RecordsLoadPlan:
 
     def load(self, driver: DBDriver) -> Optional[int]:
         plan = self.load_plan
-        return driver.load(schema=self.tbl.schema_name, table=self.tbl.table_name,
+        loader = driver.loader()
+        # TODO: Can this take in a loader so we don't have to do this assertion here?
+        assert loader is not None
+        return loader.load(schema=self.tbl.schema_name, table=self.tbl.table_name,
                            load_plan=plan, directory=self.directory)
 
     def move(self) -> MoveResult:
         logger.info(f"Connecting to database...")
+
         with self.tbl.db_engine.begin() as db:
             driver = self.tbl.db_driver(db)
+            # TODO: Can this take in a loader so we don't have to do this assertion here?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker